### ISSUE TITLE
Components should use <ng-template> instead of deprecated <template>

### ISSUE THREAD

**Issue by Nandan108** [valor-software/ngx-bootstrap#1970]
When building my project (which uses the typeahead) in aot mode, I get warnings, such as:
```
Template parse warnings:
The <template> element is deprecated. Use <ng-template> instead ("
[WARNING ->]<template [ngIf]="!isClosed">
  <div [class]="'alert alert-' + type" role="alert" [ngClass]="classes""): ...
```

Here's the list of ngx-bootstrap files containing the <template> element :
```
alert/alert.component.ts
datepicker/daypicker.component.ts
rating/rating.component.ts
sortable/sortable.component.ts
spec/ng-bootstrap/popover.spec.ts
spec/ng-bootstrap/tooltip.spec.ts
tabs/tab-heading.directive.ts
tooltip/tooltip-container.component.ts
typeahead/typeahead-container.component.ts
```
In the meantime, I executed this find/replace command to stop the warnings :
`sed -i 's/<\(\/\)\?template/<\1ng-template/g' node_modules/ngx-bootstrap/*/*`

**Comment by Martin-Luft** [comment]
Please search for same issues before opening a new one...

For Angular 4 just use the @next version which fixes this warning. This cannot be fixed in 1.6.X because it would break Angular 2 support. You can use "ngx-bootstrap": "2.0.0-beta.2" or npm install ngx-bootstrap@next

**Comment by Nandan108** [comment]
I'm sorry, but I did search... and found no open issue addressing this problem. Maybe the issue is closed?
In any case, thanks for your reply.
